libs/ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -220,14 +220,14 @@ def __init__(self, **kwargs: Any):
         super().__init__(**kwargs)
         self._client = _NVIDIAClient(
             base_url=self.base_url,
-            model=self.model,
-            default_model=self._default_model,
+            model_name=self.model,
+            default_hosted_model_name=self._default_model,
             api_key=kwargs.get("nvidia_api_key", kwargs.get("api_key", None)),
             infer_path="{base_url}/chat/completions",
         )
         # todo: only store the model in one place
         # the model may be updated to a newer name during initialization
-        self.model = self._client.model
+        self.model = self._client.model_name
```

Comment:
I am bit confused here with the usage of _NVIDIAClient.model and model_name(as both are strings)

Reply:
the idea is to have model be a Model and mode_name be a str